integration/test.ats

```diff
@@ -113,43 +124,86 @@ def tolerance_test(name, input_file, num_mpi_tasks=1, tolerance=0.00001):
     # _serac_tolerance_test_end
     """
 
-    output_dir = os.path.abspath(name) + "_output"
-    test_summary_file = "{0}/summary.json".format(output_dir)
-    baseline_summary_file = os.path.abspath("{0}.json".format(name))
+    # Set normal test variables
+    normal_output_dir    = os.path.abspath("{0}_output".format(name))
+    normal_test_file     = os.path.join(normal_output_dir, "summary.json")
+    normal_baseline_file = os.path.abspath("{0}.json".format(name))
+
+    # Set restart test variables
+    restart_output_dir    = os.path.abspath("{0}_output_restart_{1}".format(name, restart_cycle))
+    restart_test_file     = os.path.join(restart_output_dir, "summary.json")
+    restart_baseline_file = os.path.abspath("{0}_restart_{1}.json".format(name, restart_cycle))
+
+    tolerance_arg = "--tolerance={0}".format(tolerance)
+    if ".json" in str(tolerance):
+        tolerance_arg = "--tolerance-file={0}".format(tolerance)
+
+    if should_baseline:
+        # Copy pre-existing normal serac summary file to baseline location
+        copy_args = "{0} {1}".format(normal_test_file, normal_baseline_file)
+        test(
+            executable=serac_copy,
+            clas=copy_args,
+            label=name,
```

Comment:
does the output from ATS indicate that this is a baseline as opposed to a normal run?

Reply:
This is an example of the output for one copy:
`Start #   1 copy(dyn_direct_solve_serial),  nn=0, np=1, nt=0, ngpu=0 Tue Jun 28 10:57:56 2022`

It says "copy" instead of "serac" or "tolerance".